Backend/Backend/Models/Responses/DeleteUserResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Backend.Models.Responses
+{
+    public class DeleteUserResponse
+    {
+        public bool IsDeleted { get; set; }
+    }
+}
```

Comment:
Probably better have one UserResponse (where you have property with name e.x. "Status" which take Enums UserStatus ) and one Enum  UserStatus with statuses 
1. Deleted
2. Updated 
3. Failed
......

Just like advise, also we can have an Id in Response with property [JsonIgnore], when property Id == null, Response will not contain it.